tests/results/scale/edge/edge-plus.md

```diff
@@ -52,143 +52,3 @@ GKE Cluster:
 
 See [output directory](./TestScale_Listeners) for more details.
 The logs are attached only if there are errors.
-
```

Comment:
@lucacome what happened here? Did this test not run?

Reply:
I was wondering the same...

Reply:
looks like the test failed for plus: https://github.com/nginxinc/nginx-gateway-fabric/actions/runs/10001854982/job/27646232386

Shouldn't this fail the pipeline?

Reply:
It doesn't look like it. I'm adding a check in https://github.com/nginxinc/nginx-gateway-fabric/pull/2260